src/app/CMakeLists.txt

```diff
@@ -207,6 +207,10 @@ if (MUE_BUILD_MUSESAMPLER_MODULE)
     list(APPEND LINK_LIB musesampler)
 endif(MUE_BUILD_MUSESAMPLER_MODULE)
 
+if (MUE_ENABLE_AUDIO_JACK)
+    list(APPEND LINK_LIB jack)
+endif(MUE_ENABLE_AUDIO_JACK)
```

Comment:
Better move this to audio module cmake file, like https://github.com/musescore/MuseScore/blob/master/src/framework/audio/CMakeLists.txt#L273

Reply:
I see that jack is already being linked, so it's probably not needed here

```
set(MODULE_LINK ${MODULE_LINK} ${JACK_LIBRARIES} pthread )
```